addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -4,29 +4,50 @@
     <t t-name="mail.component.ChatterTopbar" owl="1">
         <div class="o_ChatterTopbar">
             <div class="o_ChatterTopbar_leftSection">
-                <button class="btn btn-link o_ChatterTopbar_buttonSendMessage">
+                <button class="btn btn-link o_ChatterTopbar_buttonSendMessage"
+                        t-att-class="{
+                            'o-buttonActive': props.isComposerVisible and props.isComposerLog === false,
+                        }"
+                        t-att-disabled="props.isDisabled"
+                        t-on-click.prevent.stop="_onClickSendMessage">
                     Send message
                 </button>
-                <button class="btn btn-link o_ChatterTopbar_buttonLogNote">
+                <button class="btn btn-link o_ChatterTopbar_buttonLogNote"
+                        t-att-class="{
+                            'o-buttonActive': props.isComposerVisible and props.isComposerLog === true,
+                        }"
+                        t-att-disabled="props.isDisabled"
+                        t-on-click.prevent.stop="_onClickLogNote">
                     Log note
                 </button>
-                <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity">
+                <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity"
+                        t-att-disabled="props.isDisabled"
+                        t-on-click.prevent.stop="_onClickSchedule">
                     <i class="fa fa-clock-o"/>
                     Schedule activity
                 </button>
                 <button class="btn btn-link o_ChatterTopbar_buttonAttachments"
+                        t-att-disabled="props.isDisabled"
                         t-on-click.prevent.stop="_onClickAttachments">
```

Comment:
inline (same remark for non-component tags, and without `t-att-class` with object variant)